R/get_pvol.R

```diff
@@ -147,6 +147,38 @@ select_get_pvol_function <- function(radar, ..., call = rlang::caller_env()) {
 }
 
 
+
```

Comment:
I'd recommend moving this to `utils.R` as the helper is called from other files as well. 

I also propose adding some simple unit tests:

- error classes
- error object parsing
- basic functionality

Just so we can check if the current behaviour is retained for future changes.

Reply:
see commits below